src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiLinkGenerator.cs

```diff
@@ -21,22 +21,25 @@ internal static class OpenApiLinkGenerator
         /// Create the collection of <see cref="OpenApiLink"/> object.
         /// </summary>
         /// <param name="context">The OData context.</param>
-        /// <param name="entitySet">The Entity Set.</param>
+        /// <param name="entityType">The Entity type.</param>
+        /// <param name ="edmTypeName">The name of the EDM type.</param>
```

Comment:
>edmTypeName [](start = 26, length = 11)

The parameter name "edmTypeName" doesn't make sense. Can we name it "sourceName" or "navigationSourceName"?

Reply:
sourceElementName